### DISCUSSION TITLE
Flipper - write sensor data log to SD-card

### DISCUSSION THREAD

**Discussion by RekordNadoev** [quen0n/unitemp-flipperzero#3] [Ideas]
Hello and thank You for this app.

I think it would be nice to have sensor data log on flipper.

Something like:
timestamp:sensor_name:temperature:humidity:pressure

**Comment by quen0n** [comment]
Hello. This feature is planned for the spring

**Reply by richpaul6806** [reply]
any update on if/when this feature will be added?
